src/components/pages/MyExpensesPage.js

```diff
@@ -1,100 +1,79 @@
-    const { t } = useTranslation();
-
-    const expenses = [
-        new Expense('Spotify', 'April 10, 2023, 11:10', '20€'),
-        new Expense('Spotify', 'April 10, 2023, 11:10', '20€'),
-        new Expense('Spotify', 'April 10, 2023, 11:10', '20€'),
-        new Expense('Spotify', 'April 10, 2023, 11:10', '20€'),
-        new Expense('Spotify', 'April 10, 2023, 11:10', '20€'),
-    ];
-
-    const amount = '100€';
-    const percentage = '+10%';
-
-    return (
-        <Wrapper>
-            <Typography variant="h1">{t('myExpenses.title')}</Typography>            
-            <ContentContainer> 
+  const { t } = useTranslation();
+  const expenses = data.expenses;
+  const amount = '100€';
+  const percentage = '+10%';
 
-                <Grid container spacing={2}>
-                    <Grid item xs={6}>
-                        <StyledCard>
-                            <StyledButton variant="text">
-                                <Typography variant="h2">{t('myExpenses.addNew')}</Typography>
-                            </StyledButton>
-                        </StyledCard>
-                    </Grid>
+  return (
+    <Wrapper>
+      <Typography variant="h1">{t('myExpenses.title')}</Typography>
+      <ContentContainer>
+        <Grid container spacing={2}>
+          <Grid item xs={6}>
+            <StyledCard>
+              <StyledButton variant="text">
+                <Typography variant="h2">{t('myExpenses.addNew')}</Typography>
+              </StyledButton>
+            </StyledCard>
+          </Grid>
 
-                    <Grid item xs={6}>
-                        <StyledCard>
-                            <TotalBalanceItem amount={amount} percentage={percentage}/>
-                        </StyledCard>
-                    </Grid>
-                </Grid>
-        
-                <Grid container spacing={1}>
-                    <Grid item xs={12}>
-                        <StyledCard>
-                            <List>
-                                {expenses.map((expense, index) => (
-                                <ExpenseItem
-                                    key={index}
-                                    name={expense.name}
-                                    timestamp={expense.timestamp}
-                                    amount={expense.amount}
-                                />
-                                ))}
-                            </List>
-                        </StyledCard>
-                    </Grid>
-                </Grid>
+          <Grid item xs={6}>
+            <StyledCard>
+              <TotalBalanceItem amount={amount} percentage={percentage} />
+            </StyledCard>
+          </Grid>
+        </Grid>
 
-            </ContentContainer>
-      </Wrapper>
-    );
-  };
\ No newline at end of file
+        <Grid container spacing={1}>
+          <Grid item xs={12}>
+            <StyledCard>
+              <List>
+                {expenses.map((expense, index) => (
```

Comment:
We should add the toggle for listing expenses or incomes. I know this PR is not for that, just leaving it here as a reminder